adoc/admin-dr-restore-master-nodes.adoc

```diff
@@ -0,0 +1,355 @@
+= Recover Master Nodes
+
+This chapter describes how to recover {productname} master nodes.
+
+== Replace Single Master Node
+
+. Remove the failed master node with skuba.
++
+Replace `<NODE_NAME>` with failed cluster master node name.
++
+----
+skuba node remove <NODE_NAME>
+----
+. Delete failed master node from `known_hosts`.
++
+Replace` <NODE_IP>` with failed master node IP address.
++
+----
+sed -i "/<NODE_IP>/d" known_hosts
+----
+. Prepare a new instance.
+. Use `skuba` to join master node from step 3.
++
+Replace `<NODE_IP>` with the new master node ip address.
++
+Replace `<NODE_NAME>` with the new master node name.
++
+Replace `<USER_NAME>` with user name.
++
+----
+skuba node join --role=master --user=<USER_NAME> --sudo --target <NODE_IP> <NODE_NAME>
+----
+
+== Recover All Master Nodes
+
+Ensure cluster version for backup/restore should be the same. Cross-version restoration in any domain is likely to  encounter data/API compatibility issues.
+
+=== Prerequisites
+
+You will only need to restore database on one of the master node (`master-0`) to regain control-plane access.
+The remaining master nodes will sync to the restored database after joining the cluster.
+
+Do the following on `master-0`. Remote restore is not supported.
+
+. Install one of the required software packages (`etcdctl`, Docker or Podman).
```

Comment:
This means that you can use ANY of these three to perform the operations, right? You only need to install one.

Reply:
Correct, we are providing 3 methods for `etcdctl` restore achieves the same goal. User can install just one of `etcdctl`, Docker or Podman.